cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -731,26 +731,18 @@ def transform_survey_responses_version_digital_delta(df: DataFrame) -> DataFrame
                     None,
                 ],
             ],
-            ["Employed", ["Employed", None, None, None]],
+            ["Employed", ["Self-employed", None, "Looking for paid work and able to start", None]],
```

Comment:
The ticket doesn't make reference to any of these changes, so I don't think they belong on this branch. May be other changes from conflict resolution. Please remove the changes if you didn't make them